src/queries/Authz.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of ciboard
+
+ * Copyright (c) 2022 Andrei Stepanov <astepano@redhat.com>
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { gql } from '@apollo/client';
+
+export const AuthZMappingQuery = gql`
```

Comment:
It is written on purpose. This is practice in securtiy world: write AuthZ and AuthN.
Since two words are similiar to stress: difference between Z an N.
Google it.

Reply:
I do not want cooperate with you.

Reply:
https://source.redhat.com/?search=authz+authn

Reply:
This naming is in conflict with the Google style guide that we agreed to adhere to. Names of variable/function/classes/… [must use](https://google.github.io/styleguide/tsguide.html#identifiers) camel case, which is [defined in the JS style guide](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined). The name should be either `AuthzMappingQuery` or `authzMappingQuery` (depending on which category you consider).

Reply:
@Andrei-Stepanov , checking the search results you provided, there are 92 mentions of `authz` and 31 mentions of `authZ`. The `authz` is used more often. Also checking github for javascript code does not show any `authZ`, but I was just checking 10 pages randomly: https://github.com/search?l=JavaScript&p=12&q=%22authZ%22&type=Code.

I would probably stick to the guidelines we agreed to use for class names.

Reply:
Update to small z.